wvd-templates/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -104,19 +104,19 @@
                 "description": "Location for all resources to be created in."
             }
         },
-        "enablePersistentDesktop": {
+        "createAvailabilitySet": {
```

Comment:
We need to wait on merging this until this change has been deployed to the WVD marketplace so we don't break re-deployments of the marketplace that points to master